tools/hyperistio/hyperistio.go

```diff
@@ -0,0 +1,156 @@
+package main
+
```

Comment:
this should move under cmd/ shouldn't it ?

Reply:
For now it's a debug tool - if it gets more usable ( including real mixer instead of mock and istioca ) - sure.